### PR TITLE
Validate instance

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Ask for confirmation if the instance doesn't exist or isn't running.

### CODE REVIEW

File: piu/cli.py [hunk 1]
```diff
@@ -200,6 +199,12 @@ def cli(ctx, config_file):
     ctx.obj = config_file
 
 
+def lookup_instance(region, ip_address):
```
Comment (tuxlife) [inline piu/cli.py]: Please move the helper function between class and cli definition. Don't mix up functions and cli-functions

File: piu/cli.py [hunk 2]
```diff
@@ -214,10 +219,12 @@ def cli(ctx, config_file):
 @click.option('--connect', help='Directly connect to the host', envvar='PIU_CONNECT', is_flag=True, default=False)
 @click.option('--tunnel', help='Tunnel to the host', envvar='PIU_TUNNEL',
               callback=tunnel_validation, metavar='LOCALPORT:REMOTEPORT')
+@click.option('--check/--no-check', help='Verify that the instance is running',
```
Comment (tuxlife) [inline piu/cli.py]: Please describe in more detail that "running" refers to the AWS status

**Review by tuxlife — COMMENTED** [review]
Unfortunately, I do not see the benefits of this change.

**Comment by aermakov-zalando** [comment]
People get better feedback instead of an obscure 'Connection timed out' error, we get less errors in the monitoring.

**Comment by tuxlife** [comment]
:+1:

**Comment by Jan-M** [comment]
Can this be reverted? Needing aws credentials and the instance checks makes piu usage less nice, slow, and fails e.g. in accounts close to rate limits.

**Comment by aermakov-zalando** [comment]
@Jan-M please see my question in #37 since we haven't heard any complaints from other users. Would you be satisfied with a config setting that would disable this?

**Pull request merged**